src/OrbitQt/servicedeploymanager.cpp

```diff
@@ -551,7 +551,17 @@ outcome::result<ServiceDeployManager::GrpcPort> ServiceDeployManager::ExecImpl()
         "running...");
   }
 
-  OUTCOME_TRY(local_grpc_port, StartTunnel(&grpc_tunnel_, grpc_port_.grpc_port));
+  outcome::result<uint16_t> local_grpc_port_result =
+      StartTunnel(&grpc_tunnel_, grpc_port_.grpc_port);
+  int retry = 3;
+  while (retry > 0 && local_grpc_port_result.has_error()) {
+    LOG("Failed to establish tunnel. Trying again in 500ms");
```

Comment:
I personally see this more as an `ERROR`.

Reply:
Fair point. I force pushed